diff --git a/node_modules/webgl-fluid-enhanced/dist/index.es.js b/node_modules/webgl-fluid-enhanced/dist/index.es.js
index 9581a63..a18b602 100644
--- a/node_modules/webgl-fluid-enhanced/dist/index.es.js
+++ b/node_modules/webgl-fluid-enhanced/dist/index.es.js
@@ -1384,7 +1384,7 @@ class D {
   calcDeltaTime() {
     const t = Date.now();
     let e = (t - this.lastUpdateTime) / 1e3;
-    return e = Math.min(e, 0.016666), this.lastUpdateTime = t, e;
+    return (e = Math.min(e, 0.1)), this.lastUpdateTime = t, e;
   }
   updateColors(t) {
     this.colorful && (this.colorUpdateTimer += t * this.colorUpdateSpeed, this.colorUpdateTimer >= 1 && (this.colorUpdateTimer = this.wrap(this.colorUpdateTimer, 0, 1), this.pointers.forEach((e) => {
